images/samba-dc/Dockerfile

```diff
@@ -20,14 +20,16 @@ ENV ADMIN_PASSWORD_SECRET=samba-admin-password \
     REALM=ad.example.com \
     SERVER_STRING="Samba Domain Controller" \
     TZ=UTC \
-    WINBIND_TRUSTED_DOMAINS_ONLY=no \
     WINBIND_USE_DEFAULT_DOMAIN=yes \
     WORKGROUP=WORKGROUP
 
 ARG BIND9_VER=9.13.2
 ARG BIND9_SHA=6c044e9ea81add9dbbd2f5dfc224964cc6b6e364e43a8d6d8b574d9282651802
 ARG SAMBA_VERSION=4.8.8-r0
 
+COPY *.conf.j2 /root/
+COPY entrypoint.sh /usr/local/bin/
```

Comment:
Those lines were moved because in some environments we had a permissions issue and must be fixed with the RUN command below.